packages/inventory/src/components/InventoryItemForm/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,276 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/InventoryItemForm renders correctly: card 1`] = `
```

Comment:
how can we reduce this snapshot?

Reply:
@p-netm I have yet found a way to reduces snapshot sizes for Antd forms without losing to assert all elements and the order of those elements...How do you go about your render test cases?